resources/views/emails/resource-digest.blade.php

```diff
@@ -1,19 +1,22 @@
 <x-mail::message>
-	# Here are the latest resources:
+# Here are the latest resources:
 
-	<x-mail::panel>
-		@foreach ($resources as $resource)
-		- [{{ $resource['name'] }}]({{ $resource['url'] }})
-		Added on {{ \Carbon\Carbon::parse($resource['created_at'])->format('F j, Y') }}
+<x-mail::panel>
+@foreach ($resources as $resource)
+- [{{ $resource['name'] }}]({{ $resource['url'] }}) <br />
+Added on {{ \Carbon\Carbon::parse($resource['created_at'])->format('F j, Y') }}
 
-		@endforeach
-	</x-mail::panel>
+@endforeach
+</x-mail::panel>
 
-	### Happy Coding!
+### Happy Coding!
 
-	### Your friends at {{ config('app.name') }}
+### Your friends at {{ config('app.name') }}
```

Comment:
I think you're right about using the word Onramp so much. What do you think of this saying: "Your friends at Tighten" or something like that

Reply:
I dig it!